garrysmod/lua/includes/extensions/util.lua

```diff
@@ -207,6 +210,33 @@ local T =
 		return self.endtime == nil or self.endtime <= CurTime()
 
 	end
+
+	--
+	-- Returns the amount of time that has passed since the Timer was started
+	--
+	GetElaspedTime = function( self )
+
+		return self:Started() and CurTime() - self.m_timestamp or 99999.9
+
+	end
+
+	--
+	-- Returns true if the amount of time that has elapsed is less than the entered duration
+	--
+	IsLessThan = function( self, duration )
+
+		return CurTime() - self.m_timestamp < duration
+
+	end
+
+	--
+	-- Returns true if the amount of time that has elapsed is greater than the entered duration
+	--
+	IsGreaterThan = function( self, duration )

```

Comment:
Why do you think these 2 are necessary?

Surely we don't need a binding to `GetElaspedTime() > 3` in the form of `IsGreaterThan( 3 )`?

Reply:
It's not necessary but helps with making the code look cleaner.
For example:
```
local timer = util.Timer()
timer:Start()

hook.Add( "Think", "TimerExample", function()

    if timer:IsGreaterThan( 3.0 ) then
    
        local ply = Entity(1)
        if IsValid( ply ) then

            ply:Kill()
            hook.Remove( "Think", "TimerExample" )
        
        end

    end

end)
```

Compared to:
```
local timer = util.Timer()
timer:Start()

hook.Add( "Think", "TimerExample", function()

    if timer:GetElaspedTime() > 3 then
    
        local ply = Entity(1)
        if IsValid( ply ) then

            ply:Kill()
            hook.Remove( "Think", "TimerExample" )
        
        end

    end

end)
```
This might not be the strongest example, but if you want the two functions removed, I will remove them.

Reply:
Yeah, I don't think this warrants adding 2 extra function aliases for a built-in operator.

I think you should remove them from this PR.